modules/example/src/main/scala/example/PostgresCopyInCsv.scala

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2013-2020 Rob Norris and Contributors
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package example
+
+import cats.implicits._
+import cats.effect.{ IO, IOApp, ExitCode }
+import doobie._
+import doobie.implicits._
+import doobie.postgres._
+import java.io.ByteArrayInputStream
+import java.nio.charset.StandardCharsets
+import java.io.InputStream
+import fs2._
+
+object PostgresCopyInCsv extends IOApp {
+
+  val xa = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver", "jdbc:postgresql://localhost/postgres", "postgres", "super-secret"
+  )
+
+  val csv = """name,food
+                |piglet,haycorns
+                |eeyore,thistles
+                |pooh,honey
+                |tigger,extract of malt""".stripMargin
+
+  // The postgres driver expects and InputStream containing the data to load
+  val is: InputStream = new ByteArrayInputStream(csv.getBytes(StandardCharsets.UTF_8))
```

Comment:
🤔 Does it matter to properly wrap this with `IO.delay`? Normally you would, because BAIS contains state, so allocating a new instance isn't referentially transparent. I'm not sure it matters enough in this example. I don't have a good sense of where people would be at, when reading the docs. If they know `IO` they'd know that they need to wrap it, and in many code samples it would end up being *probably* ok by coincidence if they don't. But it is incorrect.

Up to you I think - not a blocker to merge, at all. We could clean it after the fact if need be

Reply:
That’s probably a fair point. It doesn’t add clutter and it’s more correct.